src/shared/components/deletion-confirmation/DeletionConfirmation.scss

```diff
@@ -0,0 +1,18 @@
+@import 'src/styles/common';
+
+.deleteConfirmationContainer {
+  display: flex;
+  align-items: center;
+  justify-content: var(--container-justify-content, flex-end);
+  column-gap: var(--container-column-gap, 30px);
+  margin: var(--container-margin, 20px 0px);
```

Comment:
No; you should not set a margin on the container element here. You are building a new component, which should be adaptable to its environment as well as it can be. Margin comes from a specific context in which it is used. It is not a general property of this component.

If you want to add a margin, you will pass it set in the class name passed from the parent.

Reply:
✔